deepmd/entrypoints/test.py

```
@@ -182,24 +182,23 @@ def test(
+    if isinstance(dp, DeepPot):
+        print_ener_sys_avg(avg_err)
+    elif isinstance(dp, DeepDOS):
+        print_dos_sys_avg(avg_err)
+    elif isinstance(dp, DeepProperty):
+        print_property_sys_avg(avg_err)
+    elif isinstance(dp, DeepDipole):
+        print_dipole_sys_avg(avg_err)
+    elif isinstance(dp, DeepPolar):
+        print_polar_sys_avg(avg_err)
+    elif isinstance(dp, DeepGlobalPolar):
+        print_polar_sys_avg(avg_err)
```

Ruff (0.7.0)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)